server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -491,6 +511,20 @@ endif::[]
 
 .. The configuration of distributed cache `authenticationSessions` and other caches is left unchanged.
 
+.. It is recommended to add the `remoteStoreSecurityEnabled` property with value `false` (or eventually `true` if you enabled security
```

Comment:
(Nitpick) <pre> s/with value `false`.../with the value of `false`.../g </pre>?

Reply:
Will update. Thanks